amazon-sqs/5.11/modules/ROOT/pages/amazon-sqs-connector-reference.adoc

```diff
@@ -710,19 +744,18 @@ This operation returns a list of the queues that have the RedrivePolicy queue at
 `<sqs:list-queues>`
 
 
-This operation returns a list of your queues. The maximum number of queues that can be returned is 1000.
+Returns a list of your queues. The maximum number of queues that can be returned is 1000.
 
 
-==== Parameters
-[%header,cols="20s,20a,35a,20a,5a"]
+[%header%autowidth.spread]
 |===
 | Name | Type | Description | Default Value | Required
-| Configuration | String | The name of the configuration to use. | | x
-| Queue Name Prefix a| String |  String to use for filtering the list results. Only those queues whose name begins with the specified string are returned. |  |
-| Target Variable a| String |  The name of a variable to store the operation's output. |  |
-| Target Value a| String |  An expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  `#[payload]` |
+| Configuration | String | Name of the configuration to use. | | x
+| Queue Name Prefix a| String |  String to use for filtering the list results. Queues only with names beginning with the specified string are returned. |  |
```

Comment:
Unclear wording. I suggest:" Only queues with names that begin with the specified string are returned." or "Only returns queues with names that begin with the specified string."